javascript/javascript1/week4/VoiceAssistant.js

```diff
@@ -0,0 +1,72 @@
+const myObj = {
```

Comment:
I would recommend you to have index.html file, where you will import your js file in `<script>` tag ☺️